### DISCUSSION TITLE
Externals?

### DISCUSSION THREAD

**Discussion by ashmatash76** [Azure/ipam#315] [Q&A] [answered]
How do you add an external network in Azure IPAM?
...or endpoints on an external network?

**Comment by DCMattyG** [comment, marked as answer]
Hey @ashmatash76, happy to help!

To create an External Network, from the menu, navigate to **Externals** like so:

![image](https://github.com/user-attachments/assets/2c5994a0-eae2-4af5-931c-b667eb6de10e)

Then select a target **Space** and **Block**, followed by the menu arrow and **Add Network**:

![image](https://github.com/user-attachments/assets/b5310f63-7860-4ed6-83a4-f001399720e1)

To add **Endpoints** to the **External Network** you'll need to also create a **External Subnet**, then from the menu arrow select **Manage Endpoints**:

![image](https://github.com/user-attachments/assets/ee62cc7c-e987-4734-a75f-e2fb47564a83)

I know that is a bit high-level, but please do let me know if that help guide you in the correct direction.

**Reply by ashmatash76** [reply]
Thank you so much. Love what you are doing here, but I would have NEVER figured that out on my own :)
